ReactWindows/ReactNative/Views/Image/Composition/EffectManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Linq;
+using System.Numerics;
+using Windows.Foundation.Metadata;
+using Windows.Graphics.Effects;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Hosting;
+
+namespace ReactNative.Views.Image
+{
+    /// <summary>
+    /// Manages all Windows.UI.Composition resources.
+    /// </summary>
+    public class EffectManager : IDisposable
```

Comment:
These effect classes don't really need to be public. At least not until we find a need to use these for other modules outside of Image (outside of the core repo)

Reply:
I intentionally make it public since it can be re-used for any kind of FrameworkElement. However, since we don't have any other need for it than Image, I'll change it to internal.